benchmark/assert/throws.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const common = require('../common.js');
+const assert = require('assert');
+
+const bench = common.createBenchmark(main, {
+  n: [1e6],
+  method: [
+    'doesNotThrow',
+    'throws',
+    'throws TypeError',
```

Comment:
Would be a underscore be fine instead? The first part is the function name and that's why I'd like some kind of distinction.